### ISSUE TITLE
Is "Disable Cache" checked?

### ISSUE THREAD

**Issue by paulirish** [JonahBraun/jspm-perf-test#1]
![image](https://cloud.githubusercontent.com/assets/39191/10601400/4b2c2d50-76c6-11e5-8f22-77ca3d65c2da.png)

**Comment by JonahBraun** [comment]
No, it's not. I was testing with a hard refresh anyway to ensure a clear cache. So the 700ms without Dev Tools is not from cache.

I'm assuming the increase is incremental, which implies Dev Tools causes a 20ms penalty per XHR request (=(4000-700)/160). Doesn't that seem high?

I suppose the next step is to make a simpler test case, make a bunch of XHR requests with plain JS and see if I can reproduce the issue.

**Comment by paulirish** [comment]
20ms per XHR is unacceptable overhead, yes.

Maybe you could monitor the network activity with another third party tool (like Charles) to better visualize the before and after.

Thanks for taking a look at this... definitely want to make sure DevTools isnt slowing down your iteration

**Comment by JonahBraun** [comment]
Thanks for your sanity check on the numbers!

I don't think it's the XHR requests anymore. I wrote a XHR load test (now in this repo), loading 165 5kb JS files with simple JS. DevTools on the Network tab only caused a 2.1ms/file load time increase.

To eliminate any cause between server and browser, I wrote a simple Go web server and used it for both the jspm and XHR tests. This showed the same numbers as previously reported.

This implies that it is something else (not XHR) SystemJS is doing that is triggering the slowdown in DevTools. I'll let the SystemJS guys know and take it from there…
